dotcom-rendering/src/components/Card/Card.tsx

```diff
@@ -407,7 +407,7 @@ export const Card = ({
 	index = 0,
 	isFlexSplash,
 	showTopBarDesktop = true,
-	showTopBarMobile = false,
+	showTopBarMobile = true,
```

Comment:
Before this change, a top bar would be shown on mobile if either `showTopBarDesktop` or `showTopBarMobile` was true